### PR TITLE
Fix add/remove label issue

### PR DESCRIPTION
**Pull request by xiangpengzhao**
Should specify issue number for `GetIssueLabels`
fixes: #4921 
fixes: #4945

/cc @cjwagner @spxtr @verb

### CODE REVIEW

**Comment by cjwagner** [comment]
Oh no! Thanks for catching this!
Please also update the announcements section [here](https://github.com/kubernetes/test-infra/tree/master/prow#announcements) by changing the most recent announcement to tell users to use version 0.173 instead of 0.171.

File: prow/plugins/label/label.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) e
 	if err != nil {
 		return err
 	}
-	labels, err := gc.GetIssueLabels(org, repo, 1)
+	labels, err := gc.GetIssueLabels(org, repo, e.Number)
```
Comment (spxtr) [inline prow/plugins/label/label.go]: Wow!

**Comment by spxtr** [comment]
> Please also update the announcements section

I think that's only for breaking changes, rather than bug fixes.

**Comment by cjwagner** [comment]
> > Please also update the announcements section

> I think that's only for breaking changes, rather than bug fixes.

Yes, but shouldn't we note that 0.171 is broken and 0.173 should be used instead?
Not knowing any better, if I wanted to update my Prow instance and I looked at the announcements section I would be inclined to update hook to 0.171.
Maybe this info belongs in a version change log?

**Comment by verb** [comment]
fwiw, my Kubernetes issues were automatically moved out of the 1.9 milestone because of this bug, so there may be some clean up needed.

**Comment by cjwagner** [comment]
I'm going to LGTM this now so I can get it deployed ASAP. We can add a notice about the broken versions in another PR if that is needed.
/lgtm

**Comment by cjwagner** [comment]
> Maybe we can add a note at the top saying that it does not contain bug fixes and the like, and to keep up-to-date as best as you can?

That should be clear enough. I'll make a PR.

**Pull request merged**